kai/reactive_codeplanner/task_manager/task_manager.py

```diff
@@ -82,11 +82,6 @@ def set_seed_tasks(self, *tasks: Task) -> None:
             # Now add them back to the queue so they aren't mistakenly detected as children
             self.priority_queue.push(task)
 
-        msg = "<details><summary>Got some pending tasks</summary>"
```

Comment:
Removing this because it has too much information, the user will get overwhelmed, and the difference between a task and an incident/violation count will just be confusing.